internal/commands/commands.go

```diff
@@ -106,6 +106,13 @@ var (
 		Description: "Manage the secrets of your MongoDB Realm app",
 		Help:        "secrets",
 		SubCommands: []cli.CommandDefinition{
+			cli.CommandDefinition{
+				Use:         "create",
+				Display:     "secrets create",
+				Description: "Creates a secret for a Realm application",
```

Comment:
was this from the design team's page?  I think this should be `Realm app` according to their standards